src/components/interaction-designer/resource-editors/ResourceVariantTextEditor.vue

```diff
@@ -1,300 +1,57 @@
-      </button> -->
-    </div>
-
-    <div class="content-toolbar">
-      <!--      <block-content-autogen-button-->
-      <!--          v-if="enableAutogenButton"-->
-      <!--          :isEditable="isEditable"-->
-      <!--          :langId="langId"-->
-      <!--          :type="type"-->
-      <!--          :block="block"-->
-      <!--          class="pull-right" />-->
-
-      <span
-        v-if="isEditable"
-        class="text-muted transition-all"
-        :class="{invisible: !characterCounter.count}">
-        {{ characterCounter.count }} characters
-
-        <template v-if="mode === 'sms' && characterCounter.pages > 1">
-          ({{ characterCounter.pages }} {{ characterCounter.hasUnicode ? 'unicode pages' : 'pages' }})
-        </template>
-      </span>
-
-      <a
-        v-if="doesContentContainExpression"
-        v-b-tooltip.hover.top.html="`<p>${trans('flow-builder.youre-using-floip-expressions')}</p>
-                     <p>
-                       <strong>${trans('flow-builder.pro-tip')}:</strong>
-                       ${trans('flow-builder.floip-expressions-escape-with-double-at-symbol')}
-                     </p>`"
-        href="https://floip.gitbooks.io/flow-specification/content/fundamentals/expressions.html"
-        target="_blank">
-
-        <kbd style="margin-left: 1em">
-          <i class="glyphicon glyphicon-console" />
-          <i
-            v-if="doesContentContainExpressionError"
-            class="glyphicon glyphicon glyphicon-remove-sign text-danger" />
-          <i
-            v-else
-            class="glyphicon glyphicon-ok-sign text-success" />
-        </kbd>
-      </a>
-
-      <div
-        v-if="doesContentContainExpressionError"
-        class="alert alert-danger"
-        style="
-  margin-top: 0.5em;
-">
-        <p>
-          <i class="glyphicon glyphicon-remove-sign" />
-          <strong>
-            <a
-              href="https://floip.gitbooks.io/flow-specification/content/fundamentals/expressions.html"
-              target="_blank">FLOIP Expression</a>
-            {{ 'flow-builder.error-found' | trans }}
-          </strong>
-        </p>
-
-        <p>
-          <!-- NOTE: Funky source formatting to mitigate spaces between parens -->
-          <em>
-            {{ contentExpressionAST.message }}
-            (<span v-if="contentExpressionAST.location.start.line !== 1">{{ 'flow-builder.on-line' | trans }}
-              {{ contentExpressionAST.location.start.line }},
-            </span>{{ 'flow-builder.at-character' | trans }} {{ contentExpressionAST.location.start.column }})
-          </em>
-        </p>
-      </div>
-    </div>
-
-    <!--    <template v-if="!isEditable">-->
-    <!--      <p v-if="content">{{content}}</p>-->
-    <!--      <p>-->
-    <!--        <em class="text-muted">{{'flow-builder.no-sms-content-yet' | trans}}</em>-->
-    <!--      </p>-->
-    <!--    </template>-->
+    <expression-input
```

Comment:
btw @poojahpatil90 , do we have a future ticket to fix the autocompletion being far from the input ?
<img width="1032" alt="Screen Shot 2021-07-30 at 2 40 02 PM" src="https://user-images.githubusercontent.com/68745918/127709893-78c0e7b6-a797-4ee7-aeba-979cce1eca93.png">

Reply:
Yes, here is the ticket - https://viamoinc.atlassian.net/browse/VMO-4217
I updated the description.

Reply:
Nice, thanks